code/app.py

```diff
@@ -128,8 +76,151 @@ def alerts_page():
 
     return render_template('alerts.html')
 
-with app.app_context():
-    db.create_all()
+@app.route('/admin')
+def admin_dashboard():
+    return render_template('admin/admin-dashboard.html')
+
+@app.route('/admin-nav-bar')
+def admin_nav_bar():
+    return render_template('admin/admin-nav-bar.html')
+
+@app.route('/users')
+def get_users():
+    conn = sqlite3.connect(os.path.join(basedir, 'hurriscan.db'))
+    conn.row_factory = sqlite3.Row
+    cur = conn.cursor()
+
+    cur.execute('SELECT * FROM User')
+    rows = cur.fetchall()
+
+    users = []
+    for row in rows:
+        user = {
+            'id': row[0],
+            'username': row[1],
+            'password': row[2],
+            'email': row[3],
+            'phone': row[4],
+            'alerts_email': row[5],
+            'alerts_phone': row[6],
+            'isAdmin': row[7]
+        }
+        users.append(user)
+
+    conn.close()
+    return jsonify(users)
+
+@app.route('/delete-user/<int:user_id>')
+def delete_user(user_id):
+    try:
+        conn = sqlite3.connect(os.path.join(basedir, 'hurriscan.db'))
+        cur = conn.cursor()
+        cur.execute('DELETE FROM User WHERE id = ?', (user_id,))
+        conn.commit()
+        
+        conn.close()
+        return redirect('/admin') # Doesn't do anything, but needed to return something ¯\_(ツ)_/¯
```

Comment:
Maybe try to return null or something else instead

Reply:
thats not me thats trevor i think so I'm not touching that lol

Reply:
Ah, @trevorwinser

Reply:
I believe I tried null and it was upset with null returns.